### PR TITLE
Content 1-1-3 HelloTriangle: fixed and synchronized

### PR DESCRIPTION
**Pull request by AlexB8675**

### CODE REVIEW

File: src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp [hunk 1]
```diff
@@ -130,7 +130,8 @@ bool HelloTriangleApplication::Initialize()
             DXGI_FORMAT::DXGI_FORMAT_R32G32B32_FLOAT,
             0,
             offsetof(VertexPositionColor, position),
-            D3D11_INPUT_CLASSIFICATION::D3D11_INPUT_PER_VERTEX_DATA, 0
+            D3D11_INPUT_CLASSIFICATION::D3D11_INPUT_PER_VERTEX_DATA,
```
Comment (deccer) [inline src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp:133]: Empty line on purpose?

File: src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.hpp [hunk 2]
```diff
@@ -28,14 +28,14 @@ class HelloTriangleApplication final : public Application
     bool CreateSwapchainResources();
     void DestroySwapchainResources();
 
-bool CompileShader(
-    const std::wstring_view fileName,
-    const std::string_view entryPoint,
-    const std::string_view profile,
-    ComPtr<ID3D10Blob>& shaderBlob) const;
+    bool CompileShader(
```
Comment (deccer) [inline src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.hpp:31]: These are tabs tabs? or smart tabs? :P

**Review by deccer — CHANGES_REQUESTED** [review]
Just a question or two

File: src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.hpp (hunk 2 above)
Comment (AlexB8675) [reply src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.hpp:31]: they are spaces, I thought this wasn't aligned with the other declarations?

File: src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp (hunk 1 above)
Comment (AlexB8675) [reply src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp:133]: yes, should I revert?

Comment (deccer) [reply src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp:133]: Yes please, its part of the thing and not something separate? I might be missing something

File: src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.hpp (hunk 2 above)
Comment (deccer) [reply src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.hpp:31]: Oki doki, its not clear in this shit review screen if its spaces or tabs :) But i noticed the missing indent too :)

File: src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp (hunk 1 above)
Comment (AlexB8675) [reply src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp:133]: yeah I just though it was weird to have two different fields on the same line, reverting now

**Review by deccer — APPROVED** [review]
LGTM

**Pull request merged**